security/aws/ebs_unencrypted_volumes/AWS_Unencrypted_Volumes.pt

```diff
@@ -0,0 +1,335 @@
+name "AWS Unencrypted Volumes"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report and remediate any Elastic Block Store (EBS) volumes in AWS that are unencrypted. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/security/aws/ebs_unencrypted_volumes) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
Should remove the term "and remediate" since we're not providing any actions

Reply:
Removed the term "and remediate" .